File: src/cotestudy/완전탐색/no2309.java
package cotestudy.완전탐색;
import java.io.*;
import java.util.*;
public class no2309 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int[] intArray = new int[9];
        int sum = 0;
        int ans1 = 0;
        int ans2 = 0;
        for (int i = 0; i < 9; i++) {
            intArray[i] = Integer.parseInt(br.readLine());
            sum += intArray[i];
        }
        Arrays.sort(intArray);
        int ans = sum - 100;
        for (int i = 0; i < 8; i++) {
            for (int j = i + 1; j < 9; j++) {
                if (intArray[i] + intArray[j] == ans) {
                    ans1 = intArray[i];
                    ans2 = intArray[j];
                    break;
                }
            }
        }
        for (int i = 0; i < 9; i++) {
            if (intArray[i] != ans1 && intArray[i] != ans2) {
                bw.write(intArray[i] + "\n");
            }
        }
        bw.flush();
        bw.close();
    }
}

File: src/cotestudy/완전탐색/no2309.py
intArray = [0] * 9
sum = 0

for i in range(0,9):
    intArray[i] = int(input())
    sum += intArray[i]

intArray.sort()
ans = sum - 100
idx1 = 0
idx2 = 0
for i in range(0,9):
    for j in range(1,9):
        if intArray[i] + intArray[j] == ans:
            idx1 = i
            idx2 = j

for i in range(0,9):
    if i == idx1 or i == idx2:
        continue
    else:
        print(intArray[i])

